test/models/notification_test.rb

```diff
@@ -34,16 +34,17 @@ class NotificationTest < ActiveSupport::TestCase
   end
 
   test 'user notifications should subscribe only to itself' do
-    notification = FactoryGirl.build(:notification,
-                                     :audience => Notification::AUDIENCE_USER)
-    assert_equal [notification.initiator.id], notification.subscriber_ids
+    notification = FactoryGirl.create(:notification,
+                                      :subject => User.current,
+                                      :audience => Notification::AUDIENCE_USER)
+    assert_equal [User.current.id], notification.subscriber_ids
   end
 
   test 'usergroup notifications should subscribe to all of its members' do
     group = FactoryGirl.create(:usergroup)
     group.users = FactoryGirl.create_list(:user,25)
     notification = FactoryGirl.build(:notification,
-                                     :audience => Notification::AUDIENCE_GROUP)
+                                     :audience => Notification::AUDIENCE_USERGROUP)
     notification.subject = group
     assert group.all_users.any?
     assert_equal group.all_users.map(&:id),
```

Comment:
The IDs on these tests - they were returned in the opposite order which made one test fail (http://ci.theforeman.org/job/test_develop_pr_core/14241/database=postgresql,ruby=2.2,slave=fast/testReport/junit/(root)/NotificationTest/test_0005_Organization_notifications_should_subscribe_to_all_of_its_members/). 

Could you call `.sort` on these IDs so that they're returned in the same order? Seems that this could have happened on any of the other tests modified here too.